easybuild/framework/easyblock.py

```diff
@@ -1396,7 +1396,7 @@ def make_module_req(self):
                         self.log.info("Fixed symlink lib64 in paths for %s: %s -> %s", key, paths, fixed_paths)
                         paths = fixed_paths
                 # Use a set to remove duplicates, e.g. by having lib64 and lib which get fixed to lib and lib above
-                paths = sorted(set(paths))
+                paths = set(paths)
```

Comment:
I don't think this is the right way forward, since there's no order in a `set`.

If we want to remove duplicates, we should do something like:

```python
# remove duplicate paths
# don't use 'set' here, since order in 'paths' is important!
uniq_paths = []
for path in paths:
    if path not in uniq_paths:
        uniq_paths.append(path)
```